client/src/components/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
 import { connect } from 'react-redux';
 import * as actions from '../actions';
+import './App.css';
 import { Switch } from 'react-router';
```

Comment:
It looks like you have a music_logos.png. Are you trying to make a carousel? If so, please consider Materialize's built-in carousels rather than making your own.

Reply:
Is that cover flow? I'm not considering that. I'd like to turn play, pause, next, and previous into animated buttons later, though.

Reply:
I'm still not exactly sure what you're trying to achieve. What cover flow? This example does sound like a carousel though...